package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "ava": {
+    "babel": {
+      "presets": []
+    },
+    "concurrency": 5,
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
```

Comment:
We should add a [`"bin"` property](https://docs.npmjs.com/files/package.json.html#bin) so the tool can be called when in the `PATH`. We do this in the `create-***` packages.
Probably `dist/src/bin/markdown-link-validator.ts`